browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -150,9 +150,36 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_NOTIFICATIONS_ARE_DISABLED"  desc="Text for Notifications are disabled">
         Notifications are disabled
       </message>
+      <message name="IDS_SELECT_YOUR_COUNTRY"  desc="Text for Select your country">
+        Select your country
+      </message>
+      <message name="IDS_SELECT_YOUR_COUNTRY_DESCRIPTION"  desc="Text for Select your country so we can show you the right options and ads for your region.">
+        Select your country so we can show you the right options and ads for your region.
+      </message>
+      <message name="IDS_REWARDS_ONBOARDING_ALL_SET_DESCRIPTION"  desc="Text for Now you can get rewarded for viewing ads. You’re helping make the web a better place for everyone. And that’s awesome!">
+        Now you can get rewarded for viewing ads. You’re helping make the web a better place for everyone. And that’s awesome!
+      </message>
+      <message name="IDS_REWARDS_ONBOARDING_ALL_SET_TITLE"  desc="Woohoo You’re all set!">
+        Woohoo You’re all set!
```

Comment:
Could we make it `Woohoo! You’re all set!` to match the desktop string?

Reply:
updating it

Reply:
@mkarolin updated